.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Your workstation files are slipping in! Please have them deleted before committing! :)

Reply:
Didn't notice, I'll delete this one :)